util/nbaUtils.js

```diff
@@ -10,3 +12,26 @@ module.exports.getTeamId = function (data, firstName, lastName) {
     })
     return returnValue
 }
+
+getCurrentMonday = function() {
+    let easternTime = moment_timezone().tz('America/New_York');
+    let monDate = (moment(easternTime).day("Monday"));
+  
+    return monDate;
+  }
+  
+getDaysOfWeek = function() {

```

Comment:
getCurrentDaysInWeek ? thoughts

Reply:
Add the Monday as a parameter so it can return all the days of that current week. Future proofing for when we need the days for the next week